addon/modifiers/sortable-item.js

```diff
@@ -835,11 +842,16 @@ export default class SortableItemModifier extends Modifier {
     }
 
     if (!this.didSetup) {
-      this.addEventListener();
       this.element.dataset.sortableItem = true;
       this.sortableService.registerItem(this.groupName, this);
       this.didSetup = true;
     }
+
+    if (this.args.named.disabled && this.listenersRegistered) {
```

Comment:
If you rebase, you can do `this.named.disabled` as using `this.args` is deprecated.